lib/resty/router/router.lua

```diff
@@ -97,5 +97,38 @@ function _M:get_fields()
     return out
 end
 
+do 
+    local routers = {}
```

Comment:
We should make `routers` a weak table with `__mode = "k"`, otherwise this could leak memory.